applications/openshift/kubelet/kubelet_enable_streaming_connections/rule.yml

```diff
@@ -49,9 +54,12 @@ references:
     nist: CM-6,CM-6(1)
     srg: SRG-APP-000516-CTR-001325
 
-warnings:
-- general: |-
-    {{{ openshift_cluster_setting_kubeletconfig() | indent(4) }}}
-
 template:
-    name: kubelet_combine
+    name: yamlfile_value
+    vars:
+        filepath: '/etc/kubernetes/compliance-operator/kubeletconfig/openscap-kubeletconfig'
+        yamlpath: ".kubeletconfig.streamingConnectionIdleTimeout"
+        check_existence: "all_exist"
+        values:
+         - value: "0s"
+           operation: "not equal"
```

Comment:
I have a similar remark here.
Although item 4.2.6 from CIS v1.4.0 asks for a streaming timeout different from 0.
We can offer the experience of checking and remediating for a specific value.
Using the variable enables tailoring to the timeout value to be applied via remediations, as the checks will start failing.

Reply:
the default variable for `var_streaming_connection_timeouts` differs on some OCP version, this causes some customer cases on OCP upgrades, or when we change default value of this variable to match the OCP default, so it was left to not equal to 0. this will match benchmark expectation and also avoid this issue.